config/README.md

```diff
@@ -0,0 +1,125 @@
+| password          | The password for the repository.                      | Not set        | "mySecretPassword" | RUSTIC_PASSWORD |
+| password-file     | Path to a file containing the password for the repository. | Not set     || RUSTIC_PASSWORD_FILE |
+| password-command  | Command to retrieve the password for the repository.   | Not set        || RUSTIC_PASSWORD_COMMAND |
+| warm-up           | If true, warms up the repository by file access.      | false          ||
+| warm-up-command   | Command to warm up the repository.                    | Not set        ||
+| warm-up-wait      | The wait time for warming up the repository.          | Not set        ||
+
+### Repository Options (Additional)
+
+| Attribute         | Description                                            | Default Value  | Example Value |
+|-------------------|--------------------------------------------------------|----------------|---------------|
+| post-create-command   | Command to execute after creating a snapshot in the local backend. | Not set        | "par2create -qq -n1 -r5 %file" |
+| post-delete-command   | Command to execute after deleting a snapshot in the local backend. | Not set        | "sh -c \"rm -f %file*.par2\"" |
+
+### Snapshot-Filter Options
+
+| Attribute         | Description                                            | Default Value  | Example Value |
+|-------------------|--------------------------------------------------------|----------------|---------------|
+| filter-fn         | Custom filter function for snapshots.                 | Not set        ||
+| filter-host       | Array of hosts to filter snapshots.                   | Not set        | ["myhost"]     |
+| filter-label      | Array of labels to filter snapshots.                  | No label filter ||
+| filter-paths      | Array of paths to filter snapshots.                   | No paths filter ||
+| filter-tags       | Array of tags to filter snapshots.                    | No tags filter  ||
+
+### Backup Options
+
+| Attribute         | Description                                            | Default Value  | Example Value |
+|-------------------|--------------------------------------------------------|----------------|---------------|
+| as-path                | Specifies the path for the backup when the source contains a single path. | Not set (Source-specific option) ||
+| description            | Description for the backup.                           | Not set        ||
+| description-from       | Path to a file containing the description for the backup. | Not set    ||
+| delete-never           | If true, never delete the backup.                     | false          ||
+| delete-after           | Time duration after which the backup will be deleted. | Not set        ||
+| exclude-if-present     | Array of filenames to exclude from the backup if they are present. | Not set (Source-specific option) ||
```

Comment:
Actually this and the following options are not source-specific, but if set here, they apply for all sources (but can be overwritten in the source-specifc configuration, see below)

Reply:
I would prefer an intro text, actually...

Reply:
Yes, I would remove it and add an intro text like "The following options can be specified for each source individually in the source-individual section, see below. If they are specified here, they provide default values for all sources but can still be overwritten in the source-individual section"